pkg/operator/ceph/csi/spec.go

```diff
@@ -486,13 +486,23 @@ func startDrivers(clientset kubernetes.Interface, rookclientset rookclient.Inter
 	}
 
 	if EnableRBD {
-		err = createCSIDriverInfo(ctx, clientset, RBDDriverName)
+		rbdfsGroupPolicy, err := k8sutil.GetOperatorSetting(clientset, controllerutil.OperatorSettingConfigMapName, "CSI_RBD_FSGROUPPOLICY", "")
+		if err != nil {
+			// logging a warning and intentionally continuing with the default log level
+			logger.Warningf("failed to load CSI_RBD_FSGROUPPOLICY. %v", err)
+		}
+		err = createCSIDriverInfo(ctx, clientset, RBDDriverName, rbdfsGroupPolicy)
 		if err != nil {
 			return errors.Wrapf(err, "failed to create CSI driver object for %q", RBDDriverName)
 		}
 	}
 	if EnableCephFS {
-		err = createCSIDriverInfo(ctx, clientset, CephFSDriverName)
+		cephFSfsGroupPolicy, err := k8sutil.GetOperatorSetting(clientset, controllerutil.OperatorSettingConfigMapName, "CSI_CEPHFS_FSGROUPPOLICY", "")
```

Comment:
Nit: Go syntax guidelines suggest this should be `cephFSFsGroupPolicy`, `cephFsFsGroupPolicy`, or `cephFSFSGroupPolicy`. Trends below suggest cephFSFSGroupPolicy which is hard to read. 

`fsGroupPolicyForCephFS` is longer but avoids the weirdness.

Another option might be to create a `CSIDriverConfig` struct that could hold the driver name and fsGroupPolicy configs and allow for future config expansion without altering the `createCSIDriverInfo` function signatures. But this is definitely not necessary for just a couple configs; just an option.